exercises-python/Gui_Pereira/ex02_oo_basics/main.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+import math
+
+class Square:
+    def __init__(self, side1):
+        self.side1 = side1
+
+        self.sides = 2
+        self.area = self.side1**2
+
+    def __str__(self):
+        return f'Square {self.side1}'
+
+class Rectangle:
+    def __init__(self, side1, side2):
+        self.side1 = side1
+        self.side2 = side2
+
+        self.sides = 2
+        self.area = self.side1 * self.side2
+
+    def __str__(self):
+        return f'Rectangle {self.side1}x{self.side2}'
+
+class Circle:
+    def __init__(self, radius):
+        self.radius = radius
```

Comment:
You might want to consider making these variables private, unless they need to be exposed.